### PR TITLE
fix `-wold-style-cast`

### PR DESCRIPTION
**Pull request by FelipeLema**
continuation of https://github.com/aantron/better-enums/pull/71

### CODE REVIEW

**Comment by FelipeLema** [comment]
fixes https://github.com/aantron/better-enums/issues/48

File: enum.h [hunk 1]
```diff
@@ -14,6 +14,32 @@
...
+#         define BETTER_ENUMS_IGNORE_OLD_CAST_END _Pragma("GCC diagnostic pop")
+#      else
+#         define BETTER_ENUMS_IGNORE_OLD_CAST_HEADER
+#         define BETTER_ENUMS_IGNORE_OLD_CAST_BEGIN
+#         define BETTER_ENUMS_IGNORE_OLD_CAST_END
+#      endif
+#   endif
+#endif
+// msvc does not recognize `_Pragma`
+#ifdef _MSC_VER
```
Comment (aantron) [inline enum.h]: I think this should be `#else`, since we need the macros defined (expanding to nothing, as below) for all other compilers, not just MSVC.

Comment (FelipeLema) [reply enum.h]: just updated

Comment (aantron) [reply enum.h]: I believe you updated the wrong `#ifdef`, one of the ones below here.

Comment (FelipeLema) [reply enum.h]: woops, it should be fixed now

File: enum.h [hunk 2]
```diff
@@ -37,7 +61,7 @@
 #   endif
 #endif
 
-#ifdef _MSC_VER
+#ifdef _MSVC_VER
```
Comment (aantron) [inline enum.h]: Should be `_MSC_VER`, not `_MSVC_VER`, i.e. there should be no diff on this line.

Comment (FelipeLema) [reply enum.h]: crap!

just fixed

**Comment by aantron** [comment]
Thanks!

**Pull request merged**